locales/ru.json

```diff
@@ -104,6 +104,12 @@
       "question": "Вы действительно хотите удалить подписчика",
       "btnYes": "Да",
       "btnNo": "Нет"
+    },
+    "DeleteAvatarModal": {
+      "title": "Удалить фото профиля",
+      "question": "Вы действительно хотите удалить загруженное фото профиля",
```

Comment:
Question mark?

Reply:
it have to be in all questions or nowhere